runtime/src/accounts_db.rs

```diff
@@ -1735,6 +1792,97 @@ pub mod tests {
         assert!(check_storage(&daccounts, 2, 31));
     }
 
+    fn assert_load_account(
+        accounts: &AccountsDB,
+        slot: Slot,
+        pubkey: Pubkey,
+        expected_lamports: u64,
+    ) {
+        let ancestors = vec![(slot, 0)].into_iter().collect();
+        let (account, slot) = accounts.load_slow(&ancestors, &pubkey).unwrap();
+        assert_eq!((account.lamports, slot), (expected_lamports, slot));
+    }
+
+    fn reconstruct_accounts_db_via_serialization(accounts: AccountsDB, slot: Slot) -> AccountsDB {
```

Comment:
This is copy-n-paste thing I did in hurry. I owe you; however could you make `test_accounts_db_serialize` use this also? That change should really straight-forward.

Reply:
Yea. I can try to clean that up.